content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -156,19 +156,12 @@ If you do not specify `.spec.replicas`, then it defaults to 1.
 ### Deleting a ReplicaSet and its Pods
 
 To delete a ReplicaSet and all its pods, use [`kubectl
-delete`](/docs/reference/generated/kubectl/kubectl-commands#delete). Kubectl will scale the ReplicaSet to zero and wait
-for it to delete each pod before deleting the ReplicaSet itself. If this kubectl command is interrupted, it can
-be restarted.
-
-When using the REST API or go client library, you need to do the steps explicitly (scale replicas to
-0, wait for pod deletions, then delete the ReplicaSet).
+delete`](/docs/reference/generated/kubectl/kubectl-commands#delete). [Garbage collector](/docs/concepts/workloads/controllers/garbage-collection/) will automatically delete the dependent pods by default.
 
 ### Deleting just a ReplicaSet
 
 You can delete a ReplicaSet without affecting any of its pods, using [`kubectl delete`](/docs/reference/generated/kubectl/kubectl-commands#delete) with the `--cascade=false` option.
 
-When using the REST API or go client library, simply delete the ReplicaSet object.
```

Comment:
note the delete options to use to delete just the replicaset when using the REST API or client library (`?propagationPolicy=Orphan`)

Reply:
Done.